src/Identification/Queue/Events/Processing.php

```diff
@@ -87,9 +88,9 @@ public function __construct(JobProcessing $event)
      */
     private function unserializeToJob(string $object)
     {
-        $stdClassObj = preg_replace('/^O:\d+:"[^"]++"/', 'O:'.strlen('stdClass').':"stdClass"', $object);
+        $stdClassObj = preg_replace('/^O:\d+:"[^"]++"/', 'O:'.strlen(TenancyJob::class).':"'.TenancyJob::class.'"', $object);
```

Comment:
Instead of replacing with an StdClass, we are replacing to a TenancyJob, where we can control some things